cnxarchive/search.py

```diff
@@ -37,6 +37,7 @@
                  [chr(i) for i in range(ord('a'), ord('z') + 1)])
 WILDCARD_KEYWORD = 'text'
 VALID_FILTER_KEYWORDS = ('type', 'pubYear', 'authorID', 'submitterID')
+SEARCH_SPECIAL_CHARS = '"\',:'
```

Comment:
Hmm, currently we can search for contractions like "don't" which can actually be useful.